services/azure-servicebus/src/main/java/com/microsoft/windowsazure/services/servicebus/models/CreateEventHubResult.java

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
The current structure of requests follows this pattern and I think it would be more confusing for one model to differ on this.